livingatlas/pipelines/src/main/java/au/org/ala/pipelines/transforms/IndexFields.java

```diff
@@ -51,6 +51,7 @@ public interface IndexFields {
   String POINT_0_02 = "point-0.02";
   String POINT_0_1 = "point-0.1";
   String POINT_1 = "point-1";
+  String PRESENT_IN_COUNTRY = "presentInCountry";
```

Comment:
Wondering why we need this. Cant the data just provide countryCode ?

Reply:
Its the field name, so data will look like `presentInCountry:Australia` or `presentInCountry:Italy`. Could use country code I suppose `presentInCountry:AU` but data is from `region` field in species list, which uses full name, so would require an additional lookup.

Reply:
I've renamed the field to `taxonPresentInCountry` now.